ui/analyse/src/forecast/forecastView.ts

```diff
@@ -63,6 +65,15 @@ export default function (ctrl: AnalyseCtrl, fctrl: ForecastCtrl): VNode {
               'div.entry.text',
               {
                 attrs: dataIcon(''),
+                hook: bind('click', () => {
+                  const path = [];
+                  for (const node of nodes) {
+                    const move = parseUci(node.uci);
+                    if (!move) return;
+                    path.push(scalachessCharPair(move));
+                  }
+                  ctrl.userJump(`/?WG${path.join('')}`);
```

Comment:
Hard-coding the beginning of the path is most probably wrong, what was it for?

Reply:
I'm not actually sure just that I noticed this was extra in the actual path and missing in my computed path. 

I thought this was just a quirk of how this worked. I'll look around to see what's causing it to go missing.

Reply:
Ah, so this is the code of the moves already played on the board - which is where all forecasts start so that's why it worked

Reply:
fixed this